modules/sdk-core/src/bitgo/wallet/wallets.ts

```diff
@@ -205,6 +205,11 @@ export class Wallets implements IWallets {
       walletParams.enterprise = params.enterprise;
     }
 
+    // EVM TSS wallets must use wallet version 3
+    if ((isTss && this.baseCoin.isEVM()) !== (params.walletVersion === 3)) {
```

Comment:
What about if a wallet is TSS, but is not an EVM wallet and we wanted to use walletVersion 3? We would say

`if ( true && false ) !== true)` 
or `if (false !== true)`

So if I try to make a non-EVM TSS wallet on walletVersion 3, this will always throw an error?